docker/app/Dockerfile

```diff
@@ -1,26 +1,84 @@
-FROM sillsdev/web-languageforge:app-base-staging
+ARG ENVIRONMENT=production
+
+# APP-BASE
+FROM php:7.3.28-apache AS base-app
+
+# install apt packages
+# p7zip-full - used by LF application for unzipping lexicon uploads
+# unzip - used by LF application for unzipping lexicon uploads
+# git - TODO: why do we need git?
+# gnupg2 - necessary for LFMerge package installation via SIL sources (will be uninstalled in production)
+# curl - used by LF application
+RUN apt-get update && apt-get -y install p7zip-full unzip git gnupg2 curl
+
+# see https://github.com/mlocati/docker-php-extension-installer
+# PHP extensions required by the LF application
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+RUN install-php-extensions gd mongodb intl
+
+# install and configure LFMerge
+RUN curl -L http://linux.lsdev.sil.org/downloads/sil-testing.gpg | apt-key add - \
+&& echo "deb http://linux.lsdev.sil.org/ubuntu bionic main" > /etc/apt/sources.list.d/linux-lsdev-sil-org.list \
+&& apt-get update \
+&& apt-get install --yes --no-install-recommends python lfmerge rsyslog logrotate cron iputils-ping
+COPY docker/app/lfmerge.conf /etc/languageforge/conf/sendreceive.conf
+COPY docker/app/lfmergeqm-background.sh /
+RUN adduser www-data fieldworks \
+&& chown -R www-data:www-data /var/lib/languageforge \
+&& chmod 0755 /var/lib/languageforge \
+&& mkdir -m 02775 -p /var/www/.local \
+&& chown www-data:www-data /var/www/.local
 
 # php customizations
```

Comment:
I think we should double-check the final version of the ini that gets used...I would assume there are things in the production version that are helpful from a security perspective.